### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by Whoistolu**
# Desktop Version

In this milestone, we:
- Added the desktop version of the project
- Used media queries
- Used Flexbox and Grid layout

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Whoistolu,

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -1,545 +1,853 @@
...
+
+.desktop-header {
+  display: none;
+}
+
+.form-h3 {
+  display: none;
+}
+
+@media screen and (min-width: 992px) {
```
Comment (mke2111) [inline style.css:609]: - [ ] For desktop screen sizes, let us give equal padding/margin on the left and right of all sections.
Screenshot from 2021-08-20 14-57-57

Comment (mke2111) [inline style.css:609]: - [ ] For the desktop screens, your webpage is not responsive. The layout is not matched for all sections based on padding/margin. For this reason, there is a horizontal scroll.
Screenshot from 2021-08-20 15-01-38

Comment (mke2111) [inline style.css:609]: - [ ] You have designed your webpage for only one screen size for the mobile screens (368px). For this reason, your app is not responsive to all mobile screens. While adding in the code for the desktop version, ensure that mobile version stays intact.
Screenshot from 2021-08-20 15-06-20
Screenshot from 2021-08-20 15-08-12

File: index.html [hunk 2]
```diff
@@ -1,251 +1,285 @@
...
+                    <h4>I am a Software Developer</h4>
+                    <p>I can help you build a product, feature or website. Look<br>through some of my work and experience! If
+                        you like what<br>you see and have a project you need coded,<br>do not hesitate to contact me.</p>
+                </div>
+            </div>  
+        </section>
+
+        <section class="main-work-container">
+            <div class="work-section">
+                <h2 class="work-h1">My Recent Works</h2>
```
Comment (mke2111) [inline index.html:82]: - [ ] This heading should be placed on one line for desktop screens.
Screenshot from 2021-08-20 15-14-21

Original
Screenshot from 2021-08-20 15-15-11

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello @Whoistolu  great job working on the requested changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**